problems.py

```diff
@@ -0,0 +1,146 @@
+#1. Afișați în consolă rezultatul expresiei: 9 * (12-5) - 2**3
+print(9 * (12-5) - 2**3)
+
+#2. Realizați un nou program care calculează și afișează șapte la puterea a treia.
+print (7**3)
+
+#3. Se știe că Andrei are 7 mere. Ana are de 3 ori mai multe mere decât Andrei, iar Ștefan are de 49 de ori mai multe mere decât Andrei.
+# Să se afle câte mere au cei trei copii în total folosindu-se o expresie care conține adunarea, înmulțirea și ridicarea la putere (+,*,**).
+Andrei=7
+Ana=3*Andrei
+Stefan=49*Andrei
+print(Andrei+Ana+Stefan)
+
+#4. O mașină folosește o cantitate de x litri pentru a parcurge distanța Constanța-București.
+# O altă mașină folosește y litri pentru aceeași distanță, știind că y=2*x.
+# Să se scrie un program în care să se citească variabila x și să se determine suma necesarului de combustibil.
+def cat_combustibil (x, y):
+    x = input("Cat consuma masina ta?\n")
+    x = int (x)
+    y = x * 2
+    print(f'Daca masina ta consuma {x} litri, atunci masina mea consuma {y}\n')
+
+cat_combustibil ("x","y")
```

Comment:
try something in the lines of:
`double_int(x: int) -> int:
  return 2*x
  
 read_int(display: string)->int:
   x = input(display)
   try:
     return int(x)
   exception:
     print('Please input a number')
     
cat_combustibil():
  x = read_int("Cat consuma masina ta?\n")
  if x:
    y = double_int(x)
    print(f'Daca masina ta consuma {x} litri, atunci masina mea consuma {y}\n')
  else:
    print("x nu e numar")
`